lucien-ios/Start My Collection/StartMyCollectionViewController.xib

```diff
@@ -0,0 +1,80 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
No need to address this, but keep in mind that since your text fields have explicit height constraints, they won't be able to grow any taller if you have to add more text in the future. It won't be a problem in this project, but it might sneak up on you if you have to work on an app that's available in multiple languages (French and German for e.g. tend to be much more verbose).

Reply:
Thanks for the notice. I'll keep that in mind.